packages/access-api/test/provider-add.test.js

```diff
@@ -97,6 +99,100 @@ for (const accessApiVariant of /** @type {const} */ ([
       })
     })
   })
+
+  if (
+    ['provider/add', 'access/delegate'].every((f) =>
+      accessApiVariant.features.has(f)
+    )
+  ) {
+    it('provider/add allows for access/delegate', async () => {
+      const space = await principal.ed25519.generate()
+      const issuer = await accessApiVariant.issuer
+      const service = await accessApiVariant.audience
+      const accountDid = /** @type {const} */ ('did:mailto:example.com:foo')
+      const serviceSessionAttest = await ucanto.delegate({
+        issuer: service,
+        audience: issuer,
```

Comment:
Swapped names here are pretty confusing btw

Reply:
I agree. https://github.com/web3-storage/w3protocol/pull/492/commits/6d31d286deab582df5dde336c55f3e7368280db3

(I want to rename the `tester.issuer` to `tester.agent` or something too, but will do in standalone PR)